cmd/storj/cmd/cp.go

```diff
@@ -48,12 +48,18 @@ func copy(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	u, err := url.Parse(args[0])
+	u0, err := url.Parse(args[0])
 	if err != nil {
 		return err
 	}
 
-	if u.Scheme == "" {
+	u1, err := url.Parse(args[1])
```

Comment:
i don't know if you saw egon's change for windows support but he wrote a helper method to replace url.Parse that will work with local windows files. we should probably use that instead.